ext/standard/tests/strings/implode_error.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+--FILE--
+<?php
+/* only glue */
+try {
+    var_dump( implode("glue") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* int as pieces */
+try {
+    var_dump( implode("glue",1234) );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* NULL as pieces */
+try {
+    var_dump( implode("glue", NULL) );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* pieces as NULL array */
+try {
+    var_dump( implode(",", array(NULL)) );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* integer as glue */
+try {
+    var_dump( implode(12, "pieces") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* NULL as glue */
+try {
+    var_dump( implode(NULL, "abcd") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
```

Comment:
Usual ZPP deprecation, please remove this

Reply:
Removed